docs/maps/heatmap-layer.asciidoc

```diff
@@ -2,15 +2,12 @@
 [[heatmap-layer]]
 == Heat map layer
 
-In the heat map layer, point data is clustered to show locations with higher densities.
+In heat map layers, point data is clustered to show locations with higher densities.
 
 [role="screenshot"]
 image::maps/images/heatmap_layer.png[]
 
-You can create a heat map layer from the following data source:
-
-*Grid aggregation*:: Geospatial data grouped in grids with metrics for each gridded cell.
-Set *Show as* to *heat map*.
+To add a heat map layer to your map, click *Add layer*, then select the *Heat map* layer card.
```

Comment:
I'm wondering if here and elsewhere, you can remove the word "card".  So instead say

"then select the **Heat map** layer.

Reply:
I like it a lot better without cards. Thanks for suggesting.